modules/claims_api/app/swagger/claims_api/forms/form_0966_v1_example.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "form/0966",
+  "attributes": {
+    "type": "compensation",
+    "participant_claimant_id": 123456789,
+    "participant_vet_id": 987654321,
```

Comment:
We should still retain a minimum payload that only includes type and nothing else so that we can do a participant id look up for the vet at least.

Reply:
this was been fixed